changelog.md

```diff
@@ -4,6 +4,7 @@
 - [FIXED] Reload doesn't synchronize a null include [#4353](https://github.com/sequelize/sequelize/issues/4353)
 - [FIXED] commit/rollback multiple times on same transaction [#4491](https://github.com/sequelize/sequelize/issues/4491)
 - [FIXED] memory leak / options mangle for scopes with include [#4470](https://github.com/sequelize/sequelize/issues/4470)
+- [FEATURE] Map raw fields back to attributes when model is built [#3714](https://github.com/sequelize/sequelize/issues/3714)
```

Comment:
`[FEATURE]` > `[ADDED]`
perhaps: "Map raw fields back to attribute names when using `returning` ..."